packages/medusa/src/services/order.ts

```diff
@@ -225,6 +225,11 @@ class OrderService extends TransactionBaseService {
         : ["shipping_address", "customer"]
     }
 
+    // Remove fields related to totals from the DB select
+    config.select = (config.select ?? []).filter(
```

Comment:
question: The transformTotal does that, I suppose we are doing that to use the new total all the time right? In that case I like it, but there is other total fields and I would suggest instead to do 
```ts
const { select, relations, totalsToSelect } =
      this.transformQueryForTotals(config)
```

and just discard the totalsToSelect if that works properly so that we get both the select without the total and benefit from the new totals calculation all the time and move forward to the deprecation of it. wdyt?

Reply:
we are doing that because the remote query add these fields requesting the service `list` and they are being used to query the db and failing.
That function is not removing the fields from the select. We can talk about it.